openmrs/omod/src/main/java/org/openmrs/projectbuendia/servlet/DataExportServlet.java

```diff
@@ -89,11 +90,49 @@
     private static final int COLUMNS_PER_OBS = 3;
     private static final ClientConceptNamer NAMER = new ClientConceptNamer(Locale.ENGLISH);
 
+    public static final int DEFAULT_INTERVAL = 30;
+
+    private VisitObsValue.ObsValueVisitor stringVisitor;
+    private void defineStringVisitor() {
+        stringVisitor = new VisitObsValue.ObsValueVisitor<String>() {
```

Comment:
Style note: generally, I prefer to put implementations like this at the bottom of the class definition, and to use
```java
private final VisitObsValue.ObsValueVisitor<String> stringVisitor = new VisitObsValue.ObsValueVisitor() {
//...
};
```

Then you don't have to have the separate method definition. You could even make `stringVisitor` static and then put the name in all caps, e.g. `STRING_VISITOR`.